guides/common/modules/proc_configuring-capsule-repositories.adoc

```diff
@@ -0,0 +1,168 @@
+[id="configuring-repositories_{context}"]
+
+= Configuring Repositories
+
+Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
+
+ifdef::foreman-el,katello[]
+* xref:#repositories-centos-8[CentOS 8]
+* xref:#repositories-centos-7[CentOS 7]
+endif::[]
+ifdef::foreman-el,katello,satellite[]
+* xref:#repositories-rhel-8[Red Hat Enterprise Linux 8]
+* xref:#repositories-rhel-7[Red Hat Enterprise Linux 7]
+endif::[]
+ifdef::foreman-deb[]
+* xref:#repositories-debian-10[Debian 10 (Buster)]
+* xref:#repositories-ubuntu-2004[Ubuntu 20.04 (Focal)]
+endif::[]
+
+ifdef::foreman-el,katello[]
+== [[repositories-centos-8]]CentOS 8
```

Comment:
can you please re-sync this section with the latest version of `guides/common/modules/proc_configuring-repositories.adoc`, especially the changes made in ef2a39ee80b3438a38fe8dadc9ba9a97dfc108b1?